### DISCUSSION TITLE
Render Resolution Slider/Option for the GPU Plugin

### DISCUSSION THREAD

**Discussion by TazAileen** [runelite/runelite#18630] [Ideas]
I like having the larger draw distance but I prefer the lower resolution of the game I get with Fixed - Classic and the Stretched Mode plugin. I know it's weird, but it'd be a nice feature just in case anybody else shares my sentiment.

**Comment by skeddles** [comment]
same!!! I love the longer draw distance, but really prefer it to be more pixelated so it looks more like it used to.
